README.md

```diff
@@ -105,6 +107,13 @@ designed for use in Google's codebase. The `FluentLogger` API is recommended for
 non-Google code, since its API should remain more stable over time.
 
 <a name="more-information"></a>
+
+## Using a different logging interface
```

Comment:
Maybe say "backend" rather than "interface". Interface sort of implies "user facing logging API".

Reply:
Yeah... normally these are interfaces, but for flogger they are backends. Calling it backend can also be somewhat confusing for Java people, since there `logback` is a backend to the `slf4j` interface. Logging `middleware` maybe? Choose who you confuse :)

Reply:
Maybe say "Using a different Flogger backend" to help disambiguate?

Reply:
Done